### PR TITLE
Release 6.6.9

### PR DESCRIPTION
**Pull request by acampos1916**
**Fixes**
#906 Bugfix/905 missing token error in logs
#923 Rename GooglePay logo icon
#927 Add Affirm, Clearpay and Zip in open invoice list
#941 Implement importOrderPayment method in Adyen billing agreement

**Other**
#929 Adding Magento and Plugin versions to the bug report
#940 Add GrabPay logo

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -1067,7 +1067,10 @@ public function isPaymentMethodOpenInvoiceMethod($paymentMethod)
         if (strpos($paymentMethod, 'afterpay') !== false ||
             strpos($paymentMethod, 'klarna') !== false ||
             strpos($paymentMethod, 'ratepay') !== false ||
-            strpos($paymentMethod, 'facilypay_') !== false
+            strpos($paymentMethod, 'facilypay_') !== false ||
+            strpos($paymentMethod, 'affirm') !== false ||
+            strpos($paymentMethod, 'clearpay') !== false ||
```
Comment (acampos1916) [inline Helper/Data.php:1072]: @cyattilakiss I've been testing and could Clearpay code be `afterpaytouch` instead?

Comment (cyattilakiss) [reply Helper/Data.php:1072]: @acampos1916 afterpaytouch should be covered already with the strpos($paymentMethod, 'afterpay') !== false part.
Why?

**Review by cyattilakiss — DISMISSED** [review]

**Comment by cyattilakiss** [comment]
Pls bump version to 6.6.9 :)

File: Helper/Data.php (hunk 1 above)
Comment (acampos1916) [reply Helper/Data.php:1072]: Just that I was working with Clearpay and found out that the PM code for that is `afterpaytouch`, but you're right indeed, it is a strpos

**Review by cyattilakiss — APPROVED** [review]

**Review by peterojo — APPROVED** [review]

**Review by cyattilakiss — APPROVED** [review]

**Pull request merged**